gems/jquery-rails/CVE-2015-9251.yml

```diff
@@ -0,0 +1,64 @@
+---
+gem: jquery-rails
+cve: 2015-9251
+ghsa: rmxg-73gg-4p98
+url: https://github.com/jquery/jquery/issues/2432
+title: Cross-Site Scripting (XSS) in jquery
+date: 2018-01-22
+description: |
+  Affected versions of `jquery` interpret `text/javascript` responses from cross-origin ajax requests, and automatically execute the contents in `jQuery.globalEval`, even when the ajax request doesn't contain the `dataType` option.
+
+
+  ## Recommendation
+
+  Update to version 3.0.0 or later.
```

Comment:
Same here, text is from the jQuery CVE and mentions a different version from `jquery-rails`.